src/test/scss/parser.test.ts

```diff
@@ -324,7 +525,7 @@ suite('SCSS - Parser', () => {
 
 	test('Ruleset', function () {
 		let parser = new SCSSParser();
-		assertNode('.selector { prop: erty $let 1px; }', parser, parser._parseRuleset.bind(parser));
+		assertNode('.selector { prop: erty $let 1px m.$foo; }', parser, parser._parseRuleset.bind(parser));
```

Comment:
Same as the other orthogonal comment, just add a new one only testing `'.selector { prop: m.$foo; }'